sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj

```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure Cognitive Services Language LUIS Authoring</AssemblyTitle>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.</Description>
-    <Version>3.2.0-preview.1</Version>
+    <Version>3.8.0-preview</Version>
```

Comment:
This is quite a jump -- should this be `3.3.0-preview.1` ?

Reply:
@AlexGhiondea We got this feedback on our Runtime nuget, that when we release a preview nuget it should be the next .8 version.

Reply:
Here is the comment: https://github.com/Azure/azure-sdk-for-net/pull/5991#issuecomment-487732904

Reply:
That looks to be based on older versioning guidance. We have been updating our guidance to follow https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md. I'd suggest you updating the version to be `3.2.0.preview.2` unless there are enough changes to jump to a new minor version.